pkgs/by-name/sp/space-cadet-pinball/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchzip,
+  cmake,
+  SDL2,
+  SDL2_mixer,
+  unrar-wrapper,
+}:
+
+let
+  assets =
+    (fetchzip {
+      url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
```

Comment:
Uhhhh, is this legal? Please tell me Microsoft made these assets explicitly redistributable. I really don't want this to turn into another #306767

Reply:
I have no idea! But it was already like that so I would assume it's fine?

Reply:
cc @Atemu

Reply:
Ugh, uhh, likely not.

The package is in Nixpkgs already though with the same expressions, this is just a refactor. Let's not discuss that here.

I added it to my list of packages we probably need to remove. I'll open an issue on that at some point.